k8s/data/broker.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+import (
+	"k8s.io/client-go/1.4/pkg/api/unversioned"
+	"k8s.io/client-go/1.4/pkg/api/v1"
+)
+
+type BrokerState string
+
+const (
+	BrokerStatePending   BrokerState = "Pending"
+	BrokerStateAvailable BrokerState = "Available"
+	BrokerStateFailed    BrokerState = "Failed"
+)
+
```

Comment:
This is _slightly_ confusing to me since I naturally assumed this was the state of the broker, but it seems that it's very specifically the state of the catalog entries retrieved _from_ the broker. Any thoughts on how me might be able to make the meaning more clear?